dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/event/RumEventMapperTest.kt

```diff
@@ -164,33 +170,99 @@ internal class RumEventMapperTest {
         val mappedRumEvent = testedRumEventMapper.map(fakeRumEventCopy)
 
         // THEN
-        verify(mockLogHandler).handleLog(
+        verify(mockSdkLogHandler).handleLog(
             Log.WARN,
-            "RumEventMapper: there was no EventMapper assigned for" +
-                " RUM event type: [${fakeRumEventCopy.event.javaClass.simpleName}]"
+            RumEventMapper.NO_EVENT_MAPPER_ASSIGNED_WARNING_MESSAGE
+                .format(fakeRumEventCopy.event.javaClass.simpleName)
         )
         assertThat(mappedRumEvent).isEqualTo(fakeRumEventCopy)
     }
 
     @Test
-    fun `M return null event W map { internal mapper returns NULL }`(forge: Forge) {
+    fun `M use the original event W map returns null object { ViewEvent }`(forge: Forge) {
         // GIVEN
-        val fakeRumEvent: RumEvent = forge.getForgery()
+        val fakeBundledEvent = forge.getForgery<ViewEvent>()
+        val fakeRumEvent: RumEvent =
+            forge.getForgery<RumEvent>().copy(event = fakeBundledEvent)
+        whenever(mockViewEventMapper.map(fakeBundledEvent))
+            .thenReturn(null)
+
+        // WHEN
+        val mappedRumEvent = testedRumEventMapper.map(fakeRumEvent)
+
+        // THEN
+        assertThat(mappedRumEvent).isEqualTo(fakeRumEvent)
+        verify(mockDevLogHandler).handleLog(
+            Log.WARN,
+            RumEventMapper.VIEW_EVENT_NULL_WARNING_MESSAGE.format(fakeRumEvent.toString())
```

Comment:
very nit: I guess `toString` can be removed, `format` will call it under the hood on the `Object` type anyway.